moved/src/state_actor/tests.rs

```diff
@@ -0,0 +1,130 @@
+use {
```

Comment:
These tests are really good, but I think we should move them inside `get_block_by_number` method.

Reply:
Pushed this change